apps/ethvm/src/libs/index.ts

```diff
@@ -1,18 +1,13 @@
-import Block from '@app/libs/Block'
-import processBlocks from '@app/libs/blockProcessor'
 import { common } from '@app/libs/common'
 
-import Address from '@app/libs/Address'
 import EthValue from '@app/libs/EthValue'
 import Hash from '@app/libs/Hash'
 import Hex from '@app/libs/Hex'
 import HexNumber from '@app/libs/HexNumber'
 import HexTime from '@app/libs/HexTime'
 
 import FIFO from '@app/libs/FIFO'
-import Tx from '@app/libs/Tx'
-import processTxs from '@app/libs/txProcessor'
 
 import ethUnits from 'ethereumjs-units'
```

Comment:
Remove this import from here is not being used perse (in other srcs it's being imported directly)

Reply:
Done